### ISSUE TITLE
Object Status corners are more rounded than UI5 implementation

### ISSUE THREAD

**Issue by obc514** [SAP/fundamental-styles#3941]
The Object Status corners are more rounded than the [UI5 Implementation](https://sapui5nightly.int.sap.eu2.hana.ondemand.com/#/entity/sap.m.ObjectStatus/sample/sap.m.sample.ObjectStatus)

**Comment by droshev** [comment]
You are right! there are an update in the design spec where they removed the usage of CSS Variable `--sapElement_BorderCornerRadius` in favour of hardcoded value: 0.25rem
